src/index.html

```diff
@@ -0,0 +1,218 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+
+<body ng-app="indexModule">
+
+    <section id="page">
+
+        <header>
+
+            <hgroup>
+                <a href="index.html">
+                    <h1>Andela</h1>
+                    <h3 id="IA">Inverted Application</h3>
+                </a>
+            </hgroup>
+
+        </header>
+        <section id="articles">
+            <div class="line"></div>
+
+            <article id="article1">
+                <div class="articleBody clear">
+                    <figure>
+                        <div>
+                            <div id="uploadDiv" >
+                                <h3> Upload Your JSON file</h3>
+                                <br>
+                                <input type="file" id="fileUpload" accept=".json" multiple/>
+                                <input type="button" id="create-index" value="Create Index" />
+
+                            </div>
+                            <!--for search-->
+                            <div id="searchDiv">
+                                <h3 id="titleOption"> Serch Your JSON file</h3>
+                                <br>
+                                <div id="search-expand">
+                                    <input type="input" id="search-input" placeholder="Search Json..." />
+                                    <label> Select by names </label>
+                                        <select id="filter-drop" name="type">
+                                            <option value = "all">All</option> 
+                                        </select>
+                                <br>
+                                </div>
+                            </div>
+                        </div>
+                        <br>
+                        <div class="line"></div>
+                            <div id="showIndexId">
+                                <label> Show Index </label>
+                                <select id="index-drop-list" name="type"> </select>
+                            </div>
+                        <div class="indexDiv" id="table-holder">
+                            <table class="responstable">
+                                <thead id="table-head"></thead>
+                                <tbody id="table-body"></tbody>
+                            </table>
+                        </div>
+                    </figure>
+                </div>
+            </article>
+        </section>
+        <footer>
+            <div class="line"></div>
+            <p>Copyright 2016 - Andela Simulation - Inverted index</p>
+        </footer>
+    </section>
+    <script type="text/javascript">
+
+//  Tick present word document with jquery
+// This function populate the in index table
+let checkIndex = (dataIndex,length,fileName) => {
+    $('#table-head').append('<tr><th>'+fileName+' file</th>')
```

Comment:
- You should separate your logic from your view (have your jquery code in a separate file and include a script tag reference in your html)
- Also implement the ES6 features detailed in your tests review